carte_cli/utils/templates/dataset_flattened.md

```diff
@@ -0,0 +1,35 @@
+# {{ dataset.name }}
```

Comment:
This template basically has all the same classes and HTML elements as the MkDocs template override we've been using internally.